Ion.egg-info/requires.txt

```diff
@@ -40,21 +40,21 @@ objgraph==3.5.0
 pexpect==4.8.0
 prometheus-client==0.13.1
 psycopg2==2.9.3
-pycryptodome==3.13.0
+pycryptodome==3.14.1
 pysftp==0.2.9
 python-dateutil==2.8.2
-python-magic==0.4.24
+python-magic==0.4.25
 reportlab==3.6.7
 requests==2.27.1
-requests-oauthlib==1.3.0
+requests-oauthlib==1.3.1
 sentry-sdk==1.5.6
 setuptools-git==1.2
-Sphinx==4.3.2
-sphinx-bootstrap-theme==0.8.0
+Sphinx==4.4.0
+sphinx-bootstrap-theme==0.8.1
 vine==5.0.0
 asgiref>=3.3.4
 pillow>=9.0.0
 twisted>=21.7.0
 
 [:python_version < "3.8"]
-importlib_metadata==3.6
```

Comment:
This is to prevent `tests` from failing on `python3.7`, so we want to keep it pinned here.